accelerator/core/mam_core.c

```diff
@@ -603,8 +604,11 @@ status_t ta_send_mam_message(const ta_config_t *const info, const iota_config_t
     bundle_transactions_renew(&bundle);
     struct channel_info_s channel_info = {.ch_mss_depth = data->ch_mss_depth, .chid = data->chid};
 
+    message = (char *)malloc(sizeof(char) * (strlen(data->message) + TIMESTAMP_LEN + 1));
+    snprintf(message, TIMESTAMP_LEN + strlen(data->message) + 1, "%020ld%s", time(NULL), data->message);
```

Comment:
Is it better to change the `time()` to `clock_gettime(CLOCK_MONOTONIC)` ?

Reply:
I agree with @splasky. `time(3)` can only return the value of time in seconds. We usually need better accuracy.